src/components/about/about.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { About } from './about';
+
+describe('About Component', () => {
+  it('renders About section with correct content', () => {
+    render(<About />);
+
+    expect(screen.getByText('About')).toBeInTheDocument();
+
+    expect(screen.getByText(/Welcome to our non-profit project/)).toBeInTheDocument();
+    expect(screen.getByText(/Our goal is to provide an interactive and educational experience/)).toBeInTheDocument();
+    expect(screen.getByText(/To achieve this, we have chosen to employ the WebGPU API/)).toBeInTheDocument();
+    expect(screen.getByText(/We are excited to embark on this journey/)).toBeInTheDocument();
+
+    const link = screen.getByRole('link', { name: /List of Solar System objects by size/i });
```

Comment:
👍🏻

Reply:
It's better to use here or Regex with all lower case, or (and it's better) use just string here: { name: "List of Solar System objects by size" }